project/cmake/modules/FindVAAPI.cmake

```diff
@@ -29,6 +29,10 @@ elseif(VAAPI_INCLUDE_DIR AND EXISTS "${VAAPI_INCLUDE_DIR}/va/va_version.h")
   unset(vaapi_version_str)
 endif()
 
+if(VAAPI_VERSION_STRING VERSION_LESS 0.34)
+message(FATAL_ERROR "VAAPI minimum version 0.34 not found")
+endif()
+
 if(NOT VAAPI_FIND_VERSION)
   set(VAAPI_FIND_VERSION 0.38.0)
```

Comment:
Works perfectly fine:
-- Found VAAPI: /usr/lib/x86_64-linux-gnu/libva.so (found suitable version "0.39.0", minimum required is "0.38.0")
After changing the VAAPI_FIND_VERSION to 0.40.0 (cleaning the cache was not necessary).
-- Could NOT find VAAPI: Found unsuitable version "0.39.0", but required is at least "0.40.0" (found /usr/lib/x86_64-linux-gnu/libva.so)

There is no error thrown however because vaapi is an optional dependency, not a required one. So it'll compile without it.